site/content/en/docs/integration/toloka.md

```diff
@@ -0,0 +1,213 @@
+- [Toloka pool setup](#toloka-pool-setup)
+- [Changing Toloka pool](#changing-toloka-pool)
+- [Reviewing annotated jobs](#reviewing-annotated-jobs)
+  - [Accepting job](#accepting-job)
+  - [Rejecting job](#rejecting-job)
+- [Moving Toloka project to archive](#moving-toloka-project-to-archive)
+
+## Glossary
+
+This page contains several terms used to describe interactions between systems and actors.
+Refer to the table below for clarity on how we define and use them.
+
+<!--lint disable maximum-line-length-->
+
+| Term      | Explanation                                                                                                                                                                                                                                                                   |
+| --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Toloka    | Toloka, developed by Yandex, is a crowdsourcing platform that allows users to assign tasks to a broad group of participants, often termed "crowd workers". In the context of this article, when we mention Toloka, we are specifically referring to one of its UI interfaces. |
+| CVAT      | CVAT, developed by OpenCV, is a visualization tool designed for annotating video and image data for computer vision tasks. In this article's context, when we reference CVAT, we mean one of its UI interfaces.                                                               |
+| Requester | CVAT and Toloka user. An individual who establishes an annotation project within CVAT determines the price, creates tasks and jobs within the project, and then releases it on Toloka.                                                                                        |
+| Toloker   | Toloka user. A person who annotates the Requester's dataset using the Toloka interface.                                                                                                                                                                                       |
+
+<!--lint enable maximum-line-length-->
+
+## Preconditions
+
+Actor: Requester.
+
+Requester must have a [CVAT account](/docs/manual/basics/registration/)
+and [Toloka Requester](https://platform.toloka.ai/auth?role=REQUESTER&retpath=%2Fsignup%2Frequester)
+account.
+
+## Creating Toloka project
+
+The requester can set up a project within CVAT and subsequently
+connect it to Toloka, making it accessible for annotations by Tolokers.
+
+To initiate your Toloka project, proceed with the following steps:
+
+1. [Log in to CVAT](/docs/manual/basics/registration/#account-access)
+   and [initiate a new Project](/docs/manual/advanced/projects/).
```

Comment:
Optionally requester can setup user guide which will be shown on Toloka platform
We can just left a link to user guide documentation here

Reply:
Okay, I see this information is below. But I feel here is the better place to mention user guide